### DISCUSSION TITLE
Accessing new APIs

### DISCUSSION THREAD

**Discussion by jh3y** [johnlindquist/kit#175] [Q&A]
Not sure if my Kit is on the right version 🤔 

It's saying `1.1.0-beta.93`. Is that the version to get access to `drop`?

Check for updates gives 404 similar to mentioned in another discussion as the update/releases are ironed. Is my way forward here another manual install?

**Comment by johnlindquist** [comment]
Ah, I see. I published drafts that you can't access before you were able to get the latest release.

I'll get a public beta build out in a couple hours and dm you so you can run a "check for update".

Thanks for your patience around this. Updating/verifying/etc apps is complicated... 😬
